src/explore-education-statistics-admin/src/pages/admin-dashboard/components/MethodologySummary.tsx

```diff
@@ -111,7 +111,12 @@ const MethodologySummary = ({
                   summary={displayTitle}
                   summaryAfter={
                     <TagGroup className="govuk-!-margin-left-2">
-                      <Tag>{methodology.status}</Tag>
+                      <Tag>
```

Comment:
Looks good - do any UI tests need updating too?

Reply:
Looks like none of the UI tests need to be updated. I've added an assertion to `publish_methodology.robot` to test this change.